Dockerfile

```diff
@@ -1,54 +1,88 @@
-FROM  ubuntu:xenial
+# Use node base image
+FROM node:latest AS build
+ENV NODE_ENV=development
 
-# Add labels for metadata
+# Create a working app directory
+WORKDIR /app
+
+# Copy the package.json file into app directory to install the npm packages
+COPY ./package.json /app
+
+# Install all the npm packages
+RUN yarn install && yarn upgrade
+
+# Copy all the files of the project into the app directory
+COPY . /app
+
+# Run the yarn run build script
+RUN yarn build --pure-lockfile
```

Comment:
[`yarn install --pure-lockfile`](https://yarnpkg.com/lang/en/docs/cli/install/#toc-yarn-install-pure-lockfile)

Reply:
this should be simply `yarn build`